src/Raven.Studio/wwwroot/Content/css/pages/indexes.less

```diff
@@ -85,6 +88,22 @@
             font-size: 14px;
         }
 
+        .index-type-icon {
+            text-align: center;
+            width: 21px;
+            margin-right: @grid-gutter-width/4;
+            align-self: flex-start;
+            
+            .tooltip {
+                max-width: 400px;
+            }
+            
+            .tooltip-inner {
+                max-width: 380px;
```

Comment:
same here - tooltip inner overrides global value + ties to .tooltip style

Reply:
I changed _max-width_ to 100% for the _inner tooltip_
but for now, I am leaving this here, see above comment